custom_components/tahoma/sensor.py

```diff
@@ -362,3 +376,28 @@ def state(self):
             return self.entity_description.native_value(state.value)
 
         return state.value
+
+
+class OverkizHomeKitSetupCodeSensor(OverkizEntity, SensorEntity):
+    """Representation of an Overkiz HomeKit Setup Code."""
+
+    def __init__(self, device_url: str, coordinator: OverkizDataUpdateCoordinator):
+        """Initialize the device."""
+        super().__init__(device_url, coordinator)
+        self._attr_name = "HomeKit Setup Code"
+        self._attr_entity_registry_enabled_default = False
```

Comment:
Why hide it by default? It will be more user friendly if we show it directly, especially if some users install our component to find it.

Reply:
True, I was in doubt about this, since it will show this 'secret' in the default dashboard.

Reply:
End user can hide it just after :D